bazel/linux/templates/runner.template.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 
 set -e
+set -o pipefail
```

Comment:
nit: just do `set -euo pipefail` like in other scripts?

Reply:
I tried that first but apparently the kernel tries everything as 1 argument in shebangs... So you can't pass multiple ones. https://unix.stackexchange.com/questions/533415/invalid-option-name-error-with-shebang-bin-bash-o-pipefail-in-script